cmd/skupper/skupper.go

```diff
@@ -585,6 +597,7 @@ func NewCmdService() *cobra.Command {
 }
 
 var serviceToCreate types.ServiceInterface
+var createSvcWithGeneratedTlsCerts bool
```

Comment:
Could this field be added to types.ServiceInterface or is there a reason to separate?

Reply:
Actually, I could use `GeneratedCerts` instead, and it is already included in types.ServiceInterface. Thanks for noticing, I will update this.